edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Http/middleware/WebSocketHandlingMiddleware.cs

```diff
@@ -109,6 +135,9 @@ enum EventIds
 
             public static void WebSocketRequestNoListener(string traceId, string correlationId) =>
                 Log.LogDebug((int)EventIds.BadRequest, Invariant($"No listener found for request {traceId}. CorrelationId {correlationId}"));
+
+            public static void InvalidCertificate(Exception ex) =>
+                Log.LogWarning((int)EventIds.InvalidCertificate, Invariant($"Invalid client certificate: {ex.Message}"));
```

Comment:
Could you display the client ID? If we have several client might be useful to know the culprit.